coderd/jwt/jwe.go

```diff
@@ -0,0 +1,102 @@
+package jwt
```

Comment:
Maybe this package should be renamed to avoid conflicting with jose's jwt package name. We will need to change the import name for jose/jwt everywhere we use this